macros/base/start_print.cfg

```diff
@@ -225,9 +215,7 @@ gcode:
     #    because we already needed to use the mesh_center point as a parameter to the z calibration command
     #  - If no, then we need to use the SIZE parameter to the ADAPTIVE_BED_MESH call to do all at once now
     {% if bed_mesh_enabled %}
-        {% if status_leds_enabled %}
-            STATUS_LEDS COLOR="MESHING"
-        {% endif %}
```

Comment:
Can you put this test back ? it's mandatory to do it here...

As the file `status_leds.cfg` **is optionally included**, it's necessary to do the test **before** calling `STATUS_LEDS COLOR=xxx` to avoid an error about a non-existing macro.

Reply:
Ok done